org.uqbar.project.wollok.tests/src/org/uqbar/project/wollok/tests/interpreter/NumberTestCase.xtend

```diff
@@ -44,5 +44,177 @@ class NumberTestCase extends AbstractWollokInterpreterTestCase {
 		}
 		'''.interpretPropagatingErrors
 	}
+
+	@Test
+	def void integersBetwenTrue() {
+		'''program a {
+		
+			assert.that(3.between(1, 5))
+		
+		}
+		'''.interpretPropagatingErrors
+	}
+
+	@Test
+	def void integersBetweenFalse() {
+		'''program a {
+		
+			assert.notThat(3.between(5, 9))
+		
+		}
+		'''.interpretPropagatingErrors
+	}
+
+	@Test
+	def void absoluteValueOfAPositiveInteger() {
+		'''program a {
+		
+			assert.equals(3, 3.abs())
+		
+		}
+		'''.interpretPropagatingErrors
+	}
+
+	@Test
+	def void absoluteValueOfANegativeInteger() {
+		'''program a {
+		
+			assert.equals(3, (-3).abs())
+		
+		}
+		'''.interpretPropagatingErrors
+	}
+
+	@Test
+	def void squareRoot() {
+		'''program a {
+		
+			assert.equals(3, 9.squareRoot())
+		
+		}
+		'''.interpretPropagatingErrors
+	}
+
+	@Test
+	def void square() {
+		'''program a {
+		
+			assert.equals(9, 3.square())
+		
+		}
+		'''.interpretPropagatingErrors
+	}
+
+	@Test
+	def void lessThan() {
+		'''program a {
+		
+			assert.that(3 < 7)
+			assert.notThat(13 < 7)
+		}
+		'''.interpretPropagatingErrors
+	}
 	
+	@Test
+	def void remainder() {
+		'''program a {
+			const remainder = 17 % 7
+			const remainder2 = 12 % 6
```

Comment:
Agree. It took me 15 minutes to realize(or realise) % was modulus.
Maybe I can add a rem method (remainder remembers me old days Cobol programming)